lib/routes/account.js

```diff
@@ -887,26 +887,25 @@ module.exports = function (
             && ! doSigninConfirmation
             && emailRecord.emailVerified
           if (shouldSendNewDeviceLoginEmail) {
-            return getGeoData(ip)
-              .then(
-                function (geoData) {
-                  mailer.sendNewDeviceLoginNotification(
-                    [],
-                    emailRecord,
-                    {
-                      acceptLanguage: request.app.acceptLanguage,
-                      flowId: flowId,
-                      flowBeginTime: flowBeginTime,
-                      ip: ip,
-                      location: geoData.location,
-                      timeZone: geoData.timeZone,
-                      uaBrowser: sessionToken.uaBrowser,
-                      uaBrowserVersion: sessionToken.uaBrowserVersion,
-                      uaOS: sessionToken.uaOS,
-                      uaOSVersion: sessionToken.uaOSVersion,
-                      uaDeviceType: sessionToken.uaDeviceType
-                    }
-                  )
+            return P.all([getGeoData(ip), db.accountEmails(sessionToken.uid)])
+              .spread((geoData, emails) => {
+                mailer.sendNewDeviceLoginNotification(
+                  emails,
```

Comment:
Will `accountEmails` return the secondary email even if the feature is not enabled? I guess the `accountEmails` db call should check if the feature is enabled and return only primary if the feature is disabled.